manim/utils/config_utils.py

```diff
@@ -314,6 +331,33 @@ def _parse_cli(arg_list, input=True):
         "--config_file", help="Specify the configuration file",
     )
 
+    parser.add_argument(
+        "-q", "--quiet",
+        action="store_true",
+        help="Quiet mode. Equivalent to -v CRITICAL. It can be overridden by -v and -L",
+    )
+    parser.add_argument(
+        "-v", "--verbose",
+        type=str,
+        help="Verbosity level. Also changes the ffmpeg log level unless the latter is specified",
+        metavar="loglevel",
```

Comment:
Question: what is a metavar? What is the difference from `dest`?

Reply:
I think `metavar` is the display name when asking details through `--help`. On the other hand `dest` is the name of the attribute that will be generated.
This means that in the help message it will print `--verbose loglevel` but the attribute will be accessed through `args.verbose`.

Reply:
I would strongly prefer if the print message and the attribute be called the same.

Reply:
Okey, I removed the metavar attribute to keep the default behavior which is to show in the help message all possible values.